src/webgl/light.js

```diff
@@ -408,9 +408,21 @@ p5.prototype.pointLight = function(v1, v2, v3, x, y, z) {
  * the light is partially ambient and partially directional
  */
 p5.prototype.lights = function() {
-  this._assert3d('lights');
-  this.ambientLight(128, 128, 128);
-  this.directionalLight(128, 128, 128, 0, 0, -1);
+  if (this._colorMode !== 'rgb') { // only restore the colorMode to default if it is not in default already
```

Comment:
This line is too long which is why CI failed (not sure why it would pass locally though). I suggest moving the comment to a new line and make sure it does not exceed 80 characters long